ako-gateway-api/k8s/gateway_controller.go

```diff
@@ -137,144 +133,77 @@ func (c *GatewayController) SetupEventHandlers(k8sinfo k8s.K8sinformers) {
 	numWorkers := mcpQueue.NumWorkers
 
 	// Add EPSInformer
-	if lib.AKOControlConfig().GetEndpointSlicesEnabled() {
-		epsEventHandler := cache.ResourceEventHandlerFuncs{
-			AddFunc: func(obj interface{}) {
-				if c.DisableSync {
-					return
-				}
-				eps := obj.(*discovery.EndpointSlice)
-				namespace, _, _ := cache.SplitMetaNamespaceKey(utils.ObjKey(eps))
-				svcName, ok := eps.Labels[discovery.LabelServiceName]
-				if !ok || svcName == "" {
-					utils.AviLog.Debugf("Endpointslice Add event: Endpointslice does not have backing svc")
-					return
-				}
-				key := utils.Endpointslices + "/" + namespace + "/" + svcName
-				bkt := utils.Bkt(namespace, numWorkers)
-				c.workqueue[bkt].AddRateLimited(key)
-				utils.AviLog.Debugf("key: %s, msg: ADD", key)
-			},
-			DeleteFunc: func(obj interface{}) {
-				if c.DisableSync {
-					return
-				}
-				eps, ok := obj.(*discovery.EndpointSlice)
+	epsEventHandler := cache.ResourceEventHandlerFuncs{
```

Comment:
Same as above - probably not needed in NPL mode

Reply:
As discussed , it is needd in npl